CONTRIBUTING.md

```diff
@@ -233,3 +233,16 @@ In a rare case you may want to release only one specific package. To do so, foll
 4. Create a draft PR (not intended to be merged) to the clay repo; the sole purpose of this is to see CI green one last time before pushing the tag.
 5. Once CI is green, run `git push $REMOTE master --follow-tags`
 6. Run `yarn publish` to publish module to the npm registry.
+
+## Updating release in [liferay-portal](https://github.com/liferay/liferay-portal)
+
+1. Navigate to `./modules/apps/frontend-taglib/frontend-taglib-clay`
+2. Run `ncu '/@clayui/' -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates))
+3. Navigate to `./portal-impl` and run `ant format-source-all`
```

Comment:
I would add one to account for changes in `_components.scss` and `variables.scss` in `@clayui/css`.

So, check if those files were changed in this release, and if they were, update the overriding files in `frontend-theme-styled/clay` (I think that's the path) so that they match the updated files.

The files in `liferay-portal` are a copy of those in `clay` but including some additional imports to activate the BS3-BS4 compat layer.